libs/ui/src/lib/components/CtimsAutoCompleteComponent.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState, useEffect } from 'react';
+import { AutoComplete } from 'primereact/autocomplete';
+import axios from 'axios';
+import {getTemplate, getUiOptions, ObjectFieldTemplatePropertyType, ObjectFieldTemplateProps} from "@rjsf/utils";
+import { Tooltip } from 'antd';
+import styles from "./CtimsAutoCompleteComponent.module.css";
+
+const AutocompleteField = ({onChange, ...props }: ObjectFieldTemplateProps) => {
+      const {
+    schema,
+  } = props;
+
+  const [filteredHugoSymbols, setFilteredHugoSymbols] = useState([]);
+  const [selectedHugoSymbol, setSelectedHugoSymbol] = useState([]);
+
+  const searchSymbols = async (gene) => {
```

Comment:
do not use api calls directly in the component, create a hook where the fetching will occur

Reply:
@artonio I created a hook named useGetGenes where the fetch occurs; this is in the hooks folder. Should we also use a debounce from the Prime React library on the autocomplete component?